src/cmd/data_migrate.cpp

```diff
@@ -25,30 +29,72 @@
 /// Should be run from a folder named "persistence" consisting of the
 /// persistence
 
+#define SUCCESS 0
+#define ERROR_IN_COMMAND_LINE -1
+#define ERROR_UNHANDLED_EXCEPTION -2
+
 using namespace std;
-int main() {
+namespace po = boost::program_options;
+
+int main(int argc, const char* argv[]) {
   PairOfKey key;  // Dummy to initate mediator
   Peer peer;
+  string ignore_checker_str;
```

Comment:
ignore_checker_str default should initialize to "false"

Reply:
The bool will be passed by the comparison with the string to "true". so it's unnecessary to initialize.

Reply:
It is not a bool type, it is a string type

Reply:
Ok, i find if it is empty, ignore_checker also becomes false